src/ui/src/pages/app/dashboard/Index/index.tsx

```diff
@@ -22,45 +22,49 @@ const DashboardIndex: React.FC = () => {
             <div className="row row-cols-1 row-cols-lg-2 align-items-stretch g-4 py-2">
               <div className="col">
                 <div className="card mb-3" >
-                  <div className="row g-0" style={{height: '241px'}}>
-                    <div className="col-md-6 d-flex flex-column" style={{backgroundImage: "url('unsplash-photo-2.jpg')"}}>
-                    <div className="card card-cover text-white bg-dark rounded-5 shadow-lg" style={{height: '100%', textAlign: 'center', justifyContent: 'center'}}>
-                        <h2 className="mb-2 display-8 lh-1 fw-bold">Data Quality</h2>
-                        <a href="/monitors" className="align-items-center me-3ml-2 text-white text-decoration-none">
-                          <small>Centralize & Monitor</small>
-                          <ChevronRight className="bi me-2 pt-1" width={'1em'} height={'1em'} />
-                        </a>
+                  <a href="/monitors" className="text-decoration-none text-reset">
+                    <div className="row g-0" style={{height: '241px'}}>
+                      <div className="col-md-6 d-flex flex-column" style={{backgroundImage: "url('unsplash-photo-2.jpg')"}}>
+                      <div className="card card-cover text-white bg-dark rounded-5 shadow-lg" style={{height: '100%', textAlign: 'center', justifyContent: 'center'}}>
+                          <div className="align-items-center me-3ml-2 text-white">
+                          <h2 className="mb-2 display-8 lh-1 fw-bold">Data Quality</h2>
+                            <small>Centralize & Monitor</small>
+                            <ChevronRight className="bi me-2 pt-1" width={'1em'} height={'1em'} />
+                          </div>
+                        </div>
                       </div>
-                    </div>
-                    <div className="col-md-6 d-flex">
-                      <div className="card-body" style={{alignSelf: 'center'}}>
-                        <h4>Monitor, Alert, Analyze</h4>
-                        <p className="card-text">Get insights on the quality of your data through automatic monitoring and alerting.</p>
+                      <div className="col-md-6 d-flex">
+                        <div className="card-body" style={{alignSelf: 'center'}}>
+                          <h4>Monitor, Alert, Analyze</h4>
+                          <p className="card-text">Get insights on the quality of your data through automatic monitoring and alerting.</p>
+                        </div>
                       </div>
-                    </div>
+                    </a>
```

Comment:
It seems like there's a missing </div> tag here. When I tried running it, I couldn't start the application. Were you able to run it locally to check?